runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -687,5 +695,11 @@ public void onMenuOptionClicked(MenuOptionClicked menuOptionClicked)
 			// item spawns that are drops
 			droppedItemQueue.add(itemId);
 		}
+		// Keep queue of items placed on table
+		if (menuOptionClicked.getMenuAction() == MenuAction.ITEM_USE_ON_GAME_OBJECT)
```

Comment:
The menu action will only ever be one or the other, let's not do both checks every time
```suggestion
		else if (menuOptionClicked.getMenuAction() == MenuAction.ITEM_USE_ON_GAME_OBJECT)
```